projects/storefrontapp-e2e-cypress/cypress/integration/regression/login.e2e-spec.ts

```diff
@@ -1,22 +1,14 @@
-import { clickHamburger } from '../../helpers/homepage';
 import * as login from '../../helpers/login';
 import { viewportContext } from '../../helpers/viewport-context';
 
 describe('Login', () => {
-  viewportContext(['mobile', 'desktop'], () => {
```

Comment:
Ran the tests locally 20 times without any failures so looks good!

So I can better understand, why did we remove the desktop version of the test exactly?

Reply:
There are no substancial differences from a UI standpoint between desktop and mobile for this test. Therefore, running it in both viewports just adds execution time but it doesn't bring value.

Reply:
Makes sense!